CONFIGURATION.md

```diff
@@ -288,6 +294,34 @@ in the top-level directory for the repository.
 be expected. Your test should deposit artifacts here so `ci-operator` can expose
 them after the job has finished.
 
+## `tests.container`
+`container` is a test that runs the test commands inside a container using one
+of the images in the pipeline.
+
+## `tests.container.from`
+`from` is the pipeline image tag that this test will be run on.
+
+## `tests.openshift_ansible`
+`openshift_ansible` is a test that provisions a cluster using openshift-ansible
+and runs conformance tests.
+
+## `tests.openshift_ansible_src`
+`openshift_ansible_src` is a test that provisions a cluster using
+`openshift-ansible` and executes a command in the `src` image.
+
+## `tests.openshift_installer`
+`openshift_installer` is a test that provisions a cluster using
+`openshift-installer` and runs conformance tests.
+
+## `tests.openshift_installer_smoke`
+`openshift_installer_smoke` is a test that provisions a cluster using
+`openshift-installer` and runs smoke tests.
+
+## `tests.*.target_cloud`
+`target_cloud` can be either `aws` or `gcp` and specifies the cloud provider to
+use to provision the cluster for `openshift_ansible`, `openshift_ansible_src`,
+`openshift_installer`, and `openshift_installer_smoke` tests.
```

Comment:
Does this field have a default, or does it need to be always defined?

Reply:
All fields are required (except the one-test-type rule) and have no default value.